tests/adm_and_own_jobs_test.py

```diff
@@ -31,19 +31,22 @@ def test_adm_jobs_forbidden(auth_fc):
     from pykern import pkunit
     from pykern.pkdebug import pkdp
     from sirepo import srunit
+    import sirepo.auth_role
+    import sirepo.pkcli.roles
 
     def _op(fc, sim_type):
         with srunit.quest_start() as qcall:
             qcall.auth_db.model("UserRole").delete_all_for_column_by_values(
                 "uid",
                 [fc.sr_uid],
             )
-        r = fc.sr_post(
+        sirepo.pkcli.roles.add(fc.sr_uid, sirepo.auth_role.ROLE_USER)
```

Comment:
The delete_all_for_column_by_values is incorrect. We want to only delete the admin role.

Reply:
[4508278](https://github.com/radiasoft/sirepo/pull/7070/commits/45082783727593c14d65c391f7a7e8c66412753f)